EIPS/eip-5570.md

```diff
@@ -0,0 +1,246 @@
+---
+eip: 5570
+title: Digital Receipt Non-Fungible Tokens
+description: Non-Fungible Tokens as a method of distributing digital receipts for physical purchases, where the metadata represents a json receipt
+author: Sean Darcy (@darcys22)
+discussions-to: https://ethereum-magicians.org/t/idea-standard-digital-receipts-using-erc-721/9908
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-09-01
+requires: 721
+---
+
+## Abstract
+
+This EIP proposes a standard schema for digital receipts of transactions. Digital Receipt Non-Fungible Tokens are issued by a vendor when a customer makes a purchase from their store and contains transaction details necessary for record keeping. Digital Receipt Non-Fungible Tokens extend [EIP-721](./eip-721.md) which allows for the management and ownership of unique tokens.
+
+## Motivation
+
```

Comment:
This section doesn't answer two important questions about this proposal:

 - Why does this information need to exist on-chain?
 - If it does need to exist on-chain, why does it need to be a token (and not some other format, like a registry.)

Reply:
Awesome points. Will get some more detail here about those two

Reply:
It needs to exist for critical mass. It exhibits the greatest adaption through market place purchases. The question is a red-herring.